### DISCUSSION TITLE
How to format JavaScript code in Vue files with ace editor

### DISCUSSION THREAD

**Discussion by MrRjx** [ajaxorg/ace#5210] [Q&A]
I,m tried to format code with prettier，but it didn't achieve the expected effect

**Comment by nightwing** [comment]
Could you show, how did you try it? prettier documentation says that it should work, so maybe you are missing some configuration.

**Comment by MrRjx** [comment]
> 你能展示一下，你是如何尝试的吗？更漂亮的文档说它应该可以工作，所以你可能缺少一些配置。

thank you, I have achieved the expected effect by replaceing prettier with beautify
